Update virtual-machines-common-sizes-general.md

steveesp opened the pull request:
Remove bandwidth labels to replace them with specific max bandwidths on Dv3 series.

azurecla commented:
Hi __@GaugeField__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (abhic). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

steveesp committed:
Update virtual-machines-common-sizes-general.md

Remove bandwidth labels to replace them with specific max bandwidths on Dv3 series.
